src/api/app/models/project.rb

```diff
@@ -971,7 +968,9 @@ def add_repository_with_targets(repo_name, source_repo, add_target_repos = [], o
 
     # add repository targets
     add_target_repos.each do |repo|
-      trepo.release_targets.create(target_repository: repo, trigger: trigger)
+      unless trepo.release_targets.where(target_repository: repo).exists?
```

Comment:
it means it would create a re-release for other packages maybe. This was so far an explicit task. We could change this, but that needs discussion, so I don't want to do this here at least.